threejs/three.d.ts

```diff
@@ -526,14 +583,23 @@ declare namespace THREE {
          */
         far: number;
 
-        /**
-         * Uses focal length (in mm) to estimate and set FOV 35mm (fullframe) camera is used if frame size is not specified.
-         * Formula based on http://www.bobatkins.com/photography/technical/field_of_view.html
-         * @param focalLength focal length
-         * @param frameHeight frame size. Default value is 24.
-         */
-        setLens(focalLength: number, frameHeight?: number): void;
+        focus: number;
+        view: {
+            fullWidth: number,
+            fullHeight: number,
+            offsetX: number,
+            offsetY: number,
+            width: number,
+            height: number
+        };
+        filmGauge: number;
+        filmAcces: number;
```

Comment:
Indeed, and a big one : the property is actually `filmOffset`, instead of `filmAcces`.
Good catch !